src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -5,12 +5,23 @@
 using System.Net;
 using System.Net.Http;
 using System.Net.Tests;
-
+using System.Threading;
+using System.Threading.Tasks;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace System.Net.Http.WinHttpHandlerTests
```

Comment:
@CIPop  Could you point me to a project using the current convention please? I'm not sure what is the suggested change.